tests/pom.xml

```diff
@@ -302,20 +266,11 @@
                                 </goals>
                                 <phase>test</phase>
                                 <configuration>
-                                    <systemProperties combine.children="append">
-                                        <property>
-                                            <name>test.fpl</name>
-                                            <value>${test.ee.fpl}</value>
-                                        </property>
-                                        <property>
-                                            <name>test.health</name>
-                                            <value>health</value>
-                                        </property>
-                                        <property>
-                                            <name>test.patch.product</name>
-                                            <value>${test.patch.ee.product}</value>
-                                        </property>
-                                    </systemProperties>
+                                    <systemPropertyVariables>
```

Comment:
missing `combine.children="append"`

Reply:
I am wandering if that is really required, systemPropertyVariables seems to have this behavior by default.